.gitignore

```diff
@@ -22,4 +22,7 @@ aft
 
 #client
 /client/catalog/node_modules/
+/client/catalog/package-lock.json
```

Comment:
are we supposed to gitignore this?

Reply:
They are huge, and get created on npm install. If I don't git ignore, then explorer has a 15k long file that will be added. Seemed silly